Bottom: 7deea1c0fe70ea39c02a0b71b3223ada76bf384c
Top:    8bbd7ece6bdd78d21057bac070be0faafccfc5d5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:45 -0500

Use internal mail server


---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index eff5bcb9..272f0e98 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -41,9 +41,10 @@ container_env:
   BUILDBOT_IRC_HOST: 'chat.freenode.net'
   BUILDBOT_IRC_NICK: 'NovaTechCI'
   BUILDBOT_IRC_PASS: '{{ buildbot_irc_password }}'
-  BUILDBOT_SMTP_HOST: 'relay.dnsexit.com'
-  BUILDBOT_SMTP_USER: 'Dean.cox'
-  BUILDBOT_SMTP_PASS: 'no5156'
+  BUILDBOT_SMTP_HOST: '{{ exim4_hostname }}'
+  BUILDBOT_SMTP_PORT: '587'
+  BUILDBOT_SMTP_USER: 'buildbot'
+  BUILDBOT_SMTP_PASS: '{{ buildbot_email_password }}'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
   BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
